src/pymedx/api.py

```diff
@@ -188,11 +186,12 @@ def _get(
 
         # Set the response mode
 
-        if parameters:
-            parameters["retmode"] = output
+        parameters["retmode"] = output
 
         # Make the request to PubMed
         response = requests.get(f"{BASE_URL}{url}", params=parameters)
+        print("----------- requests")
```

Comment:
it would be good to avoid print inside the code.
alternatively, you could add a flag `verbose` with default value `False`.
and in the case verbose is True you can print logs
but if it is not really important, removing that would be a better choice, so it would be 1 less instruction to handle by the interpreter